aiohttp/helpers.py

```diff
@@ -671,15 +671,15 @@ def timeout(self) -> None:
 
 
 def ceil_timeout(delay: Optional[float]) -> async_timeout.Timeout:
-    if delay is None:
-        return async_timeout.timeout(None)
-    else:
+    if delay is not None and delay > 0:
```

Comment:
I'd prefer a guard expression here because this way you can eliminate the `else:` block entirely.
```suggestion
    if delay is None or delay <= 0:
        return async_timeout.timeout(None)
```
and then dedent the rest of this function.

Reply:
Yeah, I'd probably have done it this way if I wasn't copying the code from master. Will update the other PR for master to match.